### PR TITLE
[Improve][Connector-v2] Unset AutoCommit default to true

### PR DESCRIPTION
**Pull request by ic4y**
## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/incubator-seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/incubator-seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/incubator-seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/incubator-seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/incubator-seatunnel/blob/dev/seatunnel-dist/pom.xml)

### CODE REVIEW

**Review by TaoZex — APPROVED** [review]

File: seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/connection/SimpleJdbcConnectionProvider.java [hunk 1]
```diff
@@ -129,9 +129,6 @@ public Connection getOrEstablishConnection()
                 "No suitable driver found for " + jdbcOptions.getUrl(), "08001");
         }
 
-        //Auto commit is used by default
-        connection.setAutoCommit(true);
-
```
Comment (FWLamb) [inline seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/connection/SimpleJdbcConnectionProvider.java:134]: How can we set the `autocommit` parameter?

Comment (ic4y) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/connection/SimpleJdbcConnectionProvider.java:134]: Set in url

Comment (FWLamb) [reply seatunnel-connectors-v2/connector-jdbc/src/main/java/org/apache/seatunnel/connectors/seatunnel/jdbc/internal/connection/SimpleJdbcConnectionProvider.java:134]: However, some database implementations do not have the parameter `autocommit`, such as teradata. please track this https://teradata-docs.s3.amazonaws.com/doc/connectivity/jdbc/reference/current/frameset.html

**Review by EricJoy2048 — APPROVED** [review]

**Review by ashulin — APPROVED** [review]
+1

**Pull request merged**